data/campaigns/Eastern_Invasion/scenarios/01_Eastern_Invasion.cfg

```diff
@@ -0,0 +1,948 @@
+#textdomain wesnoth-ei
+
+#
+# This is intended to be a difficult mission, moreso than the next few.
+# I think high difficulty is necessary to establish the undead (and particularly Mal-Ravanal) as a credible threat.
+#
+# At first, this scenario is pretty easy. The enemies have low gold and low income.
+# But once Mal-Ravanal shows up, things get dangerous:
+#    1) this (hopefully) makes the player hate Mal-Ravanal right off the bat
+#    2) this establishes the undead invasion as a credible threat.
+#
+# In my expeience, the best strategy is to push forward and gather XP in the first half,
+# then give ground after Mal-Ravanal shows up, during the second night.
+# You need to have enough units/hp to push forward during the third day.
+#
+
+[scenario]
+    id=01_Eastern_Invasion
+    name= _ "Eastern Invasion"
+    map_file=01_Eastern_Invasion.map
+    next_scenario=02_The_Escape_Tunnel
+    turns=11
+    victory_when_enemies_defeated=no
+
+    {DEFAULT_SCHEDULE_DUSK}
+
+    {SETUP_CAMPFIRES}
+
+    {INTRO_AND_SCENARIO_MUSIC main_menu.ogg loyalists.ogg}
+    {LET_INTRO_MUSIC_FINISH}
+    [story]
+        [part]
+            # It is 625 YW. Konrad II is king, having reigned since 612.
+            story= _ "It was the thirteenth year of Konrad II’s reign when the strange occurrences in the Estmark Hills on the eastern border of Wesnoth began."
+            background=story/S01-1.jpg
+        [/part]
+        [part]
+            story= _ "Cattle and beasts of burden were found dead in the fields and villagers vanished from their houses with nary a clue nor sign of struggle. At first, the settlers suspected raiders from the great desert, but when they sent scouts east they found nothing but a small mountain range and miles of swamp."
+            background=story/S01-2.jpg
+        [/part]
+        [part]
+            story= _ "The disappearances did not stop and every day the terror grew greater. Eventually, the people living in the eastern villages sent a messenger to the King, asking for help. The messenger rode hard for many days and nights and finally reached the city of Weldyn."
```

Comment:
Is it worth having a comma after 'nights', if only to separate 'rode hard for many days and nights' from 'finally reached the city'?

Reply:
You mean after 'stop'? That one seems okay to me, but I don't mind either way. The main reason I queried the last sentence was because without a comma there's two unrelated 'and's in a row.

Reply:
they're different types of "ands" though; I just read "days and nights" as a single unit, so I didn't even notice that it had an "and" in it... but yeah I guess adding a comma there is fine...

Reply:
And the comma is to clarify that.

Reply:
* I agree on a comma after "stop"
* Another possibility on the end is: "For many days and nights, finally reaching…"